docs/user-documentation/glossary.md

```diff
@@ -1,95 +1,202 @@
+
+## PR
+See [Pull request](#pull-request)
+
+## Pull request
+Also PR; sometimes also known as merge requests; technical term from distributed version control systems for software code like Git. Code contributors can request that the maintainer of a code repository 'pulls' the code change into the repository after approval.
+
+## RDF
+See [Resource Description Framework](#resource-description-framework)
+
+## Repository Item
+See: [Resource Node](#resource-node)
+
+## Resource Description Framework
+Also RDF; family of World Wide Web Consortium (W3C) specifications originally designed as a data model for metadata. It has come to be used as a general method for conceptual description or modeling of information that is implemented in web resources. The data is modelled as a set of statements, also known as triples. A collection of RDF statements intrinsically represents a directed graph. Data represented according to the RDF specifications can be serialized in different ways, for instance using [JSON-LD](#json-ld).
+
 ## Resource Node
-A resource node is a Drupal node that represents a single conceptual item or object stored in an Islandora repository. It acts as a container for all files and metadata associated with that item, and is the place where the item 'lives' as a visitable url. 
+A resource node is a [Drupal](#drupal) [Node](#node) that represents a single conceptual item or object stored in an Islandora repository. It acts as a container for all files and metadata associated with that item, and is the place where the item 'lives' as a visitable URI. 
  
-For example, A video stored in Islandora will have a resource node, which metadata stored in fields, a preservation master file, and may have a thumbnail, web-friendly derivatives, and technical metadata associated with the resource node. The resource node may also belong to one or more collections. 
+For example, a video stored in Islandora will have a resource node, with metadata stored in [Fields](#field). Attached to the resource node is a [Media](#media) entity, which encapsulates the preservation-grade file. The Resource node may be linked to further [Media](#media), for instance for a thumbnail, web-friendly derivative, and technical metadata associated with the resource node. The resource node may also belong to one or more collections. 
+
+## Taxonomy
  
 ## Taxonomy term
-A term used to classify content, such as a tag or a category. 
+In the context of Islandora, a taxonomy term is a [Drupal](#drupal) entity of the type 'taxonomy'. [Drupal](#drupal) generally uses taxonomy terms to classify content (tag or category). In Islandora, taxonomies are used to establish controlled vocabularies for describing resources, for instance for standardised spellings of names or subject terms.
```

Comment:
**Should be changed for accuracy:** The Drupal entity type is actually "taxonomy term" (as per https://www.drupal.org/docs/user_guide/en/planning-data-types.html) "Taxonomy" is the core Drupal module that adds taxonomy term and vocabulary functionality. 

RE: "In Islandora, taxonomies are used to establish controlled vocabularies..." (For clarity/precision, refer to "taxonomy vocabularies" rather than "taxonomies", and ideally move to entry for taxonomy vocabulary and explain that while you can use taxonomy vocabularies in Drupal as you would a controlled vocabulary, Drupal offers zero functionality for actually controlling that vocabulary.) **Or ignore this paragraph altogether and I can work on it later, as I have significant additions to make to the Metadata page on this topic! :-)**

Reply:
Good point. The Drupal documentation isn't exactly consistent on this, unfortunately. I cribbed from https://www.drupal.org/docs/drupal-apis/entity-api/bundles, where they seem to refer to the entity type as "Taxonomy" (and talk about the notion of Bundles, which I personally find more confusing than helpful). The page that you linked to is newer, and referring to the entity type as 'taxonomy terms' also makes more sense. I'll revise the entry accordingly. If you have more to add at a later point, feel free to jump in as well.

Reply:
The lack of consistency in terminology used in the Drupal documentation drives me bonkers!! I will likely have an update to some of the taxonomy stuff in the new year. Thanks!